cli/internal/codegen/testdata/expected_golang.go

```diff
@@ -76,28 +76,28 @@ func WithHTTPClient(client HTTPDoer) Option {
 	}
 }
 
-// WithAuthToken allows you to set an authentication token to be used for each request.
-//
-// This token will be sent as a Bearer token in the Authorization header.
-func WithAuthToken(bearerToken string) Option {
+// WithAuth allows you to set the authentication data to be used with each request
```

Comment:
I swapped the Go client gen from a `token` auth to a data structure auth, so there's more changes in this file than you might expect. But this is fine